.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
```

Comment:
What does this file do?

Reply:
I have the same question, I looked into this PR briefly, I thought the new rules are OK, but I wasn't sure about additional changes in this PR.

Reply:
no to cherry-picking.  Where will these tests run?  Will they work for us?  Do we need them?  Arew we using Swift packages?  Can we disable them?

Reply:
I don't think we need to do anything. As the author mentions in the [1.0.0 release](https://github.com/airbnb/swift/releases/tag/1.0.0), we would need to add a spm dependency to use it

Reply:
ok.  monitor the github actions and see what happens.

Reply:
thanks for the clarification. I thought it would not work work regardless, given we were not adding the spm dependency.

Should I add it to a git ignore so it doesn't get added in future syncs, given we are diverging from airbnb?

Reply:
I agree with @ccorneliu about removing this file, as it was confusing to me too. However if this is more work, I'm ok to keep it.